src/components/ebay-fake-tabs/README.md

```diff
@@ -20,10 +20,11 @@
 
 ## ebay-fake-tabs Attributes
 
-| Name             | Type   | Stateful | Required | Description                                                             |
-| ---------------- | ------ | -------- | -------- | ----------------------------------------------------------------------- |
-| `selectedIndex`  | String | Yes      | No       | 0-based index of selected tab tab and panel                             |
-| `tabAriaCurrent` | Enum   | No       | No       | If provided, will set aria-current to provided value. Default is "page" |
+| Name                                                | Type    | Stateful | Required | Description                                                                                                       |
+| --------------------------------------------------- | ------- | -------- | -------- | ----------------------------------------------------------------------------------------------------------------- |
+| `selectedIndex`                                     | String  | Yes      | No       | 0-based index of selected tab tab and panel                                                                       |
+| `tabMatchesCurrentUrl`                              | Boolean | No       | No       | If provided, will set aria-current to "page" if true, or will set aria-current "true" if value provided is false. |
+| default is true and will set aria-current to "page" |
```

Comment:
Markdown syntax error here? Let's go with:

"Specify whether the href of the currently active fake tab matches the current window url. Default is true. This property is used to configure the underlying aria-current attribute (i.e. a value of "page" (default) or "true").